MAUI/Calendar/events.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title: Events in .NET MAUI Calendar control | Syncfusion
+description: Learn here all about the Events support in Syncfusion .NET MAUI Calendar (SfCalendar) control and more details.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Events in .NET MAUI Calendar (SfCalendar)
+
+The Calendar has the events `ViewChanged`, `SelectionChanged`, `Tapped`, `DoubleTapped` and `LongPressed` for notifying after user interactions in [.NET MAUI Calendar]. 
```

Comment:
The Calendar has the events `ViewChanged,` `SelectionChanged,` `Tapped,` `DoubleTapped,` and `LongPressed` for notifying after user interactions in [.NET MAUI Calendar].

Reply:
Content modified